pngcanvas.py

```diff
@@ -47,6 +47,16 @@ def gradient_list(start, end, steps):
             for i in range(steps + 1)]
 
 
+def rgb2rgba(rgb):
+    """Take a row of RGB bytes, and convert to a row of RGBA bytes."""
+    rgba = []
+    for i in range(0, len(rgb), 3):
+        rgba += rgb[i:i+3]
+        rgba.append(255)
+    assert len(rgba) == len(rgb) / 3 * 4
```

Comment:
What exactly is the concern with using `assert`? I can remove it, but I don't think the performance advantage of removing it is really worth it. I think it adds some useful documentation, and helps to make sure any modifications to the function internals don't break stuff.

But if it is a big concern I can just remove it.